modules/feature_extraction/random_forest_feature_extractor.py

```diff
@@ -25,7 +25,9 @@ def __init__(self, samples, cluster_indices, n_splits=10, n_iterations=10, scali
 
     def train(self, train_set, train_labels):
         # Construct and train classifier
-        classifier = RandomForestClassifier(n_estimators=self.n_estimators, n_jobs=self.njobs)
+        classifier = RandomForestClassifier(n_estimators=self.n_estimators, 
+                                            n_jobs=self.njobs,
```

Comment:
Making it possible to use a fixed seed for reproducability, just as for MLP, and settings njobs to use all processors